redaxo/src/core/lib/minibar/minibar_element.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ *
+ * @package redaxo\core
+ */
+abstract class rex_minibar_element
+{
+    /**
+     * Returns the html bar item
+     *
+     * @return string
+     */
+    abstract public function serve();
```

Comment:
Vllt besser „render“? serve finde ich in diesem context unpassend

Reply:
Akzeptiert, die Drinks wurden serviert … Elemente werden gerendert :)

Reply:
Hehe, ahhh daher wehte der wind :-)

Reply:
das war schon alles durchdacht 😄

Reply:
ist angepasst db195e89d4dcc0947d10ecdb011fccb4c71bfb95